CONVENTIONS.md

```diff
@@ -189,6 +189,9 @@ following taints if doing so is necessary to form a functional Kubernetes node:
 Operators should not specify tolerations in their manifests for any of the taints in
 the above list without an explicit and credible justification.
 
+Operators should never specify the following toleration:
```

Comment:
The other guidelines in this section include justification, so that someone coming to the doc without all of the background can learn why we have each restriction. This addition would benefit from a sentence or two explaining why this value is so special that it should never be used.

Reply:
Thanks, that looks good.